src/mixer.c

```diff
@@ -312,38 +312,39 @@ void MIXER_ApplyMixer(struct Mixer *mixer, volatile s32 *raw, s32 *orig_value)
     value = value * mixer->scalar / 100 + PCT_TO_RANGE(mixer->offset);
 
     //4th: multiplex result
+    s32 scaled_value = raw[mixer->dest + NUM_INPUTS + 1];
     switch(MIXER_MUX(mixer)) {
     case MUX_REPLACE:
         break;
     case MUX_MULTIPLY:
-        value = raw[mixer->dest + NUM_INPUTS + 1] * value / CHAN_MAX_VALUE;
+        value = scaled_value * value / CHAN_MAX_VALUE;
         break;
     case MUX_ADD:
-        value = raw[mixer->dest + NUM_INPUTS + 1] + value;
+        value = scaled_value + value;
         break;
     case MUX_MAX:
-        value = raw[mixer->dest + NUM_INPUTS + 1] > value
-                  ? raw[mixer->dest + NUM_INPUTS + 1]
+        value = scaled_value > value
+                  ? scaled_value
                   : value;
         break;
     case MUX_MIN:
-        value = raw[mixer->dest + NUM_INPUTS + 1] < value
-                  ? raw[mixer->dest + NUM_INPUTS + 1]
+        value = scaled_value < value
+                  ? scaled_value
                   : value;
         break;
     case MUX_DELAY:
         {
             //value initially represents 20ths of seconds to cover 60-degrees
             //convert value to represent #msecs to cover 60-degrees (zero->full)
             if (value == 0 || orig_value == NULL) {
-                value = raw[mixer->dest + NUM_INPUTS + 1];
+                value = scaled_value;
                 break;
             }
-            value = abs(RANGE_TO_PCT(value)) * 50; //'50' represents 1/20 of a second (1000msec/20 = 50)
+            value = abs(RANGE_TO_PCT(value)) * 50;
             //rate represents the maximum travel per iteration (once per MEDIUM_PRIORITY_MSEC)
```

Comment:
Please restore the comment.  I put it there because I couldn't recall where the 50 came from in the 1st place